drivers/hmis/app/graphql/types/hmis_schema/ce_referral.rb

```diff
@@ -73,16 +73,16 @@ def client
     end
 
     def current_steps
-      load_ar_association(object, :current_steps)
+      load_ar_association(object, :current_steps).sort_by(&:available_at)
```

Comment:
If parallel steps became available at the same time, this might not be deterministic - recommend secondary sort by ID